frames/src/main/java/com/checkout/frames/screen/billingaddress/billingaddressdetails/models/BillingAddress.kt

```diff
@@ -14,4 +14,20 @@ internal data class BillingAddress(
         "",
         Address("", "", "", "", "", Country.from(Locale.getDefault().country))
     )
+
+    internal companion object {
+        internal val DEFAULT_BILLING_ADDRESS by lazy { BillingAddress() }
+
+        /**
+         * Whether the [BillingAddress] has been edited or same as the default one.
+         */
+        internal fun BillingAddress.isEdited() = (name == DEFAULT_BILLING_ADDRESS.name &&
```

Comment:
What exactly does the `DEFAULT_BILLING_ADDRESS.name` have in it?

Reply:
it's a empty string given the current implementation:

```
internal constructor() : this(
    "",
    Address("", "", "", "", "", Country.INVALID_COUNTRY)
)
```

Reply:
Thanks for logging it Chintan!